package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "rnw-front-end-development",
-  "version": "0.0.1",
+  "version": "0.3.1",
```

Comment:
This wasn't done in the past ... is this right?

Reply:
It is, I always forget to bump the version there